lib/utils/isAfterSingleLineComment.mjs

```diff
@@ -0,0 +1,19 @@
+import isSharedLineComment from './isSharedLineComment.mjs';
+
+/**
+ * @param {import('postcss').Node} node
+ * @returns {boolean}
```

Comment:
[note] Specifying a return type makes code clearer as documentation. Also, it's valid for type-check.